doc/source/project/organization.rst

```diff
@@ -153,6 +154,9 @@ Pull requests changing or clarifying the *Procedure Documents* (listed above):
   contain code modifications.
 - If the pull request adds or removes *committers*, there must be a separate
   pull request for each person added or removed.
+- Members of the *committers* group may submit PRs updating their own name in
+  the ``members`` section of this file.  This type of change is not subject to
+  the 192 hour nor 25% rule, and can be merged by any other member.
```

Comment:
I'm gonna be that person, sorry, does "their name" means their human name and explicitly not their username? Or both? 
(I'm in favour of allowing members to update both human + usernames.)

Reply:
I have a feeling we may need to be a bit more careful with GitHub username changes, but agreed it'd be to good to clarify which ones are allowed by this change.

Reply:
Agreed. For the simple case (rename without claiming the old name) it's probably sufficient to make the approval as long as the old username redirects to the new one, or at least if some of the repos do?

Reply:
According to https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/changing-your-github-username , there's no redirect from the old profile page to new username's one, only for repositories.

Reply:
My intent was to allow users to change both; I'm happy to reopen with "updating their own name or username" instead if folks want to do that.

Re: being careful -- there's no linkage to this plain text file to any permissions or anything, so I'm not *that* worried about that.  (Right?  I could change my github name to @yoplait tomorrow and would have the same permissions regardless of what's in this file)

Reply:
> there's no linkage to this plain text file to any permissions or anything

if we wanted there to be we could just link to the github team.

Reply:
> if we wanted there to be we could just link to the relevant github team within galaxyproject. Thereby avoiding keeping a document up to date with respect to the actual situation

Sure, that should work. We would need to change the procedure to add/remove people from the team though, since it would not be possible to use a PROCEDURES pull request.

Reply:
> My intent was to allow users to change both; I'm happy to reopen with "updating their own name or username" instead if folks want to do that.

@dannon I prefer that wording, let's do that.

Reply:
@hexylena Sure thing; I'll close this and reopen with changes per procedure.